docs/pages/typography.js

```diff
@@ -0,0 +1,1008 @@
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">20sp</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="400">and so</Text>
+                </Table.Cell>
+              </Table.Row>
+              <Table.Row>
+                <Table.Cell>
+                  <Text size="200">500</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">28px</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">28pt</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">28sp</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="500">are</Text>
+                </Table.Cell>
+              </Table.Row>
+              <Table.Row>
+                <Table.Cell>
+                  <Text size="200">600</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">28px</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">28pt</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="200">28sp</Text>
+                </Table.Cell>
+                <Table.Cell>
+                  <Text size="600">you</Text>
+                </Table.Cell>
+              </Table.Row>
+            </Table.Body>
+          </Table>
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Line length"
+          description="For long-form, multi-line paragraphs set at our default font size, use a maximum width of 664 to allow for a max of about 90 characters. Otherwise, it can get hard for users to scan and read text."
+        >
+          <Box width="100%" color="infoWeak">
+            <LineLength />
+          </Box>
+        </MainSection.Subsection>
+      </MainSection>
+      <MainSection name="Spacing">
+        <MainSection.Subsection
+          title="Line height"
+          description="Proper line spacing line height (or leading) is important for readability. If lines of text get too close together, it can be hard to read them; if they are too far apart it can make it difficult to sense a clear hierarchy and group related text blocks. Given that we use system fonts, we rely on percentages on web and system defaults on mobile to ensure blocks are text are still readable for internationalization and dynamic sizing."
+        />
+        <MainSection.Subsection
+          title="Web"
+          description="We currently use browser defaults on web UIs so that lines of text are readable in all
+            languages and scripts. For example, scripts like Hindi have larger ascenders and
+            descenders than Latin scripts. Setting a fixed line height can make scripts like Hindi
+            hard to read."
```

Comment:
I think you can delete all this (duplicated below)

Reply:
Thanks! Done!